2021-07-01 13:18:46,367 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 13:18:46,368 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:18:46,368 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:18:46,368 INFO    MainThread:14764 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131846-27shpfdn\logs\debug.log
2021-07-01 13:18:46,369 INFO    MainThread:14764 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131846-27shpfdn\logs\debug-internal.log
2021-07-01 13:18:46,370 INFO    MainThread:14764 [wandb_init.py:init():370] calling init triggers
2021-07-01 13:18:46,370 INFO    MainThread:14764 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 13:18:46,371 INFO    MainThread:14764 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 1hgp63xi
2021-07-01 13:18:46,373 INFO    MainThread:14764 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/1hgp63xi
2021-07-01 13:18:46,374 INFO    MainThread:14764 [jupyter.py:save_history():339] not saving jupyter history
2021-07-01 13:18:46,374 INFO    MainThread:14764 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-01 13:18:46,375 INFO    MainThread:14764 [wandb_init.py:_jupyter_teardown():269] cleaning up jupyter logic
2021-07-01 13:18:46,375 INFO    MainThread:14764 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-01 13:18:46,376 INFO    MainThread:14764 [wandb_run.py:_restore():1566] restore
2021-07-01 13:18:47,042 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 55464
  total_bytes: 55464
}

2021-07-01 13:18:47,397 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 55464
  total_bytes: 55464
}

2021-07-01 13:18:48,020 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 55464
  total_bytes: 59457
}

2021-07-01 13:18:48,355 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 55464
  total_bytes: 59457
}

2021-07-01 13:18:48,457 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:48,561 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:48,663 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:48,765 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:48,867 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:48,969 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:49,071 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 59457
  total_bytes: 59457
}

2021-07-01 13:18:50,207 INFO    MainThread:14764 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-01 13:18:50,209 INFO    MainThread:14764 [wandb_run.py:_show_history():1909] rendering history
2021-07-01 13:18:50,212 INFO    MainThread:14764 [wandb_run.py:_show_files():1938] logging synced files
2021-07-01 13:18:50,223 INFO    MainThread:14764 [wandb_init.py:init():419] starting backend
2021-07-01 13:18:50,223 INFO    MainThread:14764 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-01 13:18:50,230 INFO    MainThread:14764 [backend.py:ensure_launched():135] starting backend process...
2021-07-01 13:18:50,395 INFO    MainThread:14764 [backend.py:ensure_launched():140] started backend process with pid: 13712
2021-07-01 13:18:50,396 INFO    MainThread:14764 [wandb_init.py:init():424] backend started and connected
2021-07-01 13:18:50,427 INFO    MainThread:14764 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-01 13:18:50,427 INFO    MainThread:14764 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-01 13:18:50,427 INFO    MainThread:14764 [wandb_init.py:init():472] updated telemetry
2021-07-01 13:18:50,429 INFO    MainThread:14764 [wandb_init.py:init():491] communicating current version
2021-07-01 13:18:53,248 INFO    MainThread:14764 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-01 13:18:53,248 INFO    MainThread:14764 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-01 13:18:53,701 INFO    MainThread:14764 [wandb_init.py:init():529] starting run threads in backend
2021-07-01 13:18:54,118 INFO    MainThread:14764 [wandb_run.py:_console_start():1624] atexit reg
2021-07-01 13:18:54,120 INFO    MainThread:14764 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-01 13:18:54,121 INFO    MainThread:14764 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-01 13:18:54,123 INFO    MainThread:14764 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-01 13:18:54,124 INFO    MainThread:14764 [wandb_init.py:init():554] run started, returning control to user process
2021-07-01 13:18:57,344 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-01 13:19:47,576 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-01 13:19:54,167 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-01 13:20:06,128 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 13:20:06,128 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:20:06,128 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:20:06,129 INFO    MainThread:14764 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_132006-126zoa7f\logs\debug.log
2021-07-01 13:20:06,129 INFO    MainThread:14764 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_132006-126zoa7f\logs\debug-internal.log
2021-07-01 13:20:06,130 INFO    MainThread:14764 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F187BFAF28>
2021-07-01 13:20:06,131 INFO    MainThread:14764 [wandb_init.py:init():370] calling init triggers
2021-07-01 13:20:06,131 INFO    MainThread:14764 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 13:20:06,131 INFO    MainThread:14764 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 27shpfdn
2021-07-01 13:20:06,133 INFO    MainThread:14764 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/27shpfdn
